CourseGrab/Home/HomeTableViewHeader.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  HomeTableViewHeader.swift
+//  CourseGrab
+//
+//  Created by Mathew Scullin on 2/4/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class HomeTableViewHeader: UITableViewHeaderFooterView {
+
+    private let title = UILabel()
+
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+
+        contentView.backgroundColor = .white
+        title.font = ._20Semibold
+        contentView.addSubview(title)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupContraints (section: Int) {
+        let padding = section == 0 ? 24 : 6
+        title.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(20)
+            make.bottom.equalToSuperview().inset(6)
+            make.top.equalToSuperview().inset(padding)
+        }
+    }
+
+    func configure(section: Int, numSections: Int, count: Int, totalCount: Int) {
+        setupContraints(section: section)
```

Comment:
i don't think a separate function is needed here since we only call it once. lets just move it over into this configure function